tests/manager/celery/tasks/test_search.py

```diff
@@ -73,6 +75,36 @@ def test_search_reindex(
     end_to_end_search_fixture.expect_results([work1, work2, work4], "", ordered=False)
 
 
+def test_PP_1332_fiction_returns_results(
```

Comment:
I don't think the test name here has to refer to a ticket. It would be nice to have the test name stand on its own, so we don't have to go digging in jira in the future when we are refactoring this test.